compute_endpoint/tests/unit/test_status_reporting.py

```diff
@@ -28,7 +28,7 @@ def test_status_reporting(engine_type, engine_runner, engine_heartbeat: float):
 
     # Confirm heartbeats in regular intervals
     for _i in range(3):
-        q_msg = results_q.get(timeout=1)
+        q_msg = results_q.get(timeout=1.1)
```

Comment:
If you were seeing this timeout getting hit at 1, I would recommend putting in 2s as the timeout. We often see the tests running on some old machines which fail tests with close timing expectations.

Reply:
Good call.